nextcord/member.py

```diff
@@ -656,6 +656,29 @@ async def kick(self, *, reason: Optional[str] = None) -> None:
         Kicks this member. Equivalent to :meth:`Guild.kick`.
         """
         await self.guild.kick(self, reason=reason)
+        
+    async def timeout(self, timeout: Union[datetime.datetime, datetime.timedelta],
+                      *, reason: Optional[str] = None):
```

Comment:
This function formatting is inconsistent with other methods in this file, could it be changed to match them please.
```suggestion
    async def timeout(
        self,
        timeout: Union[datetime.datetime, datetime.timedelta],
        *,
        reason: Optional[str] = None,
    ) -> None:
```